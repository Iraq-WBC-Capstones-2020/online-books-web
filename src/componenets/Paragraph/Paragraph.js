import React from 'react';
import {MDBBox } from "mdbreact";

export default function AuthorPage() {
    return (
        <MDBBox tag='p' className='lead'>
        Vivamus sagittis lacus vel augue laoreet rutrum faucibus dolor auctor.
        Duis mollis, est non commodo luctus.
      </MDBBox>
    );
}
