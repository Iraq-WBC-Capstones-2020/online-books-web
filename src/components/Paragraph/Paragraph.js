import React from 'react';

function Paragraph ({ title, children }) {
  return (
    <>
      <p className="text-3xl font-medium border-b-4 inline border-blue-500">
        {title}
      </p>
      <p className="text-base leading-loose text-justify">{children}</p>
    </>
  );
};

export default Paragraph;
