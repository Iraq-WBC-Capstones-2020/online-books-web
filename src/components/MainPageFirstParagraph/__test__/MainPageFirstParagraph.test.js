import React from 'react';
import renderer from 'react-test-renderer';
import Paragraph from './../MainPageFirstParagraph';

test('MainPageFirstParagraph snapshot', () => {
  const tree = renderer.create(<Paragraph />).toJSON();
  expect(tree).toMatchSnapshot();
});
