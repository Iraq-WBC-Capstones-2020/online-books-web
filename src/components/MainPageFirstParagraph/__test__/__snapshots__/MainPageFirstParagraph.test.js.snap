// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MainPageFirstParagraph snapshot 1`] = `
Array [
  <div
    className="container"
    data-test="container"
  >
  <p className="font-serif text-center text-black font-bold my-12 text-xl bgLightblue">
      YOU ARE NOT LOST YOU ARE HERE
    </p>
   
]
`;
